docs/program-setup/bare_metal/baremetal_usage.md

```diff
@@ -0,0 +1,75 @@
+# Using bare metal profile
+
+## Creating a bare metal application
```

Comment:
I will change all of this to numbered steps. This is easier for users when they have to go do something and then come back to a document.